src/electos/ballotmaker/cli.py

```diff
@@ -12,6 +14,10 @@
 STYLE_HELP = "Stylesheet file for ballot generation"
 VERSION_HELP = "Print the version number."
 
+# configure logging
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
```

Comment:
If you use `sys.stdout` redirected regular output can't be separated from logging output. The default stream is `sys.stderr` to avoid this. (The Logging HowTo [What happens if no configuration is provided](https://docs.python.org/3.8/howto/logging.html#what-happens-if-no-configuration-is-provided) describes this behavior, though it's not completely clear about the rationale.)

If you are doing this for a deliberate reason, that's fine, I am just flagging this in case you aren't.

Reply:
This is a good point, and I completely agree, but at this point in the development cycle I'm in Debug mode so I want **everything** to show up on stdout. When it's time to ship this, the logging level will go to `stderr`, level ERROR by default. In addition, I'd like to add some command line options to set the logging destination (`stdout` or a log file) and level to whatever you want to use at runtime. I'll add this as an issue, in fact.

Reply:
Issue created: [Implement CLI options for logging destination, level · Issue #69 · TrustTheVote-Project/BallotLab](https://github.com/TrustTheVote-Project/BallotLab/issues/69)

Nice!